apps/dsp-app/cypress.config.ts

```diff
@@ -5,6 +5,9 @@ export default defineConfig({
   e2e: {
     specPattern: 'cypress/**/**/**/*.cy.ts',
     excludeSpecPattern: ['*.spec.js', '*.spec.ts'],
+    viewportHeight: 768,
```

Comment:
why do we setup this ? Can't we have it flexible to have multiple test with different sizes ?

Reply:
I dont like default cypress size, its even smaller than this and not targeted to our users, also easier to see in the runner. I believe it can be overridden or pass different configuration files for multiple resolution testing.